portal/src/main/webapp/css/header.css

```diff
@@ -149,6 +149,18 @@ header {
     width:100%;
 }
 
+.legacy .modal-header .close span {
+    display:none;
+}
+
+.legacy .modal-header .close:before {
+    content: "✖";
```

Comment:
is this definitely not going to break? maybe should be an html format, like &something;